modules/microshift-storage-backup-volume-snapshots.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// microshift/volume-snapshots-microshift.adoc
+
+:_content-type: PROCEDURE
+[id="backing-up-a-volume-snapshotting_{context}"]
+= Backing up a volume snapshot
+
+Snapshots of {product-title} data are created as read-only logical volumes (LVs) located on the same devices as the original data. All local volumes must be manually mounted before they can be copied as persistent volumes (PVs) and used as backup copies. To use a snapshot of a {product-title} storage volume as a backup, you can locate it on the local host and then move it to a secure location. To find specific snapshots and copy them, use the following procedure.
```

Comment:
> Snapshots of {product-title} data are created 

It's not MicroShift's data being snapshotted, it's the user's application data. Phrased the way it is, a reader may mistakenly think we are taking backups of the MicroShift database via this process.